tools/make_dist_html.py

```diff
@@ -114,7 +112,7 @@ def groups_for(groups, n=4):
 
 if __name__ == '__main__':
   os.chdir(os.path.dirname(__file__))
-  tags = subprocess.check_output(['git', 'tag'])
+  tags = subprocess.check_output(['git', 'tag'], text = True )
```

Comment:
```suggestion
  tags = subprocess.check_output(['git', 'tag'], text=True)
```
I'll commit that before merging.